index.html

```diff
@@ -12,8 +12,164 @@
 
 <body>
 
-<h1>test Font</h1>
-<p>Lorem ipsum dolor sit amet.</p>
+<header>
+    <!--Nav staat los van het outer en inner container principe. Nav houdt wel de breedte -->
+    <nav class="outer-container">
+        <ul class="nav-links-container">
+            <li><a href="#portfolio">Case Studies</a></li>
+            <li><a href="#services">Services</a></li>
+            <li><a href="#contact">Contact Us</a></li>
+        </ul>
+    </nav>
+    <!--    Outer en inner container principe begint bij dit deel van de header-->
+    <div class="outer-container">
+        <div class="inner-container-sm header-cta">
+            <h1>Your website is an investment in the success of your business.
+            </h1>
+            <p class="header-subtitle">We are here to help you succeed.</p>
+            <button type="button">Lets' work together</button>
+            <img class="squiggle-line-stripe" src="assets/squiggle_line_stripe.svg"
+                 alt="squiggle-line-stripe">
+        </div>
+    </div>
+</header>
+<div class="header-triangle"></div>
 
+<section class="hello outer-container">
+    <div class="inner-container-sm">
+        <div class="hello-content-container">
+            <h2>Rekupe is a digital agency in Los Angeles.
+            </h2>
+            <h2>We use strategic UX, SEO optimization and beautiful visual
+                designs to help your business grow.
+            </h2>
+            <h2>We see every project as a fresh opportunity, pushing
+                ourselves to
+                create new digital experiences.
+            </h2>
+            <p>Have a question? <a href="/">Click here</a> to contact us!</p>
```

Comment:
Een klein detail, maar wanneer we naar een niet-bestaande pagina refereren, gebruiken we niet een slash, maar een hashtag.